recipes/libxml2/all/conanfile.py

```diff
@@ -17,6 +17,7 @@ class Libxml2Conan(ConanFile):
     # from ./configure and ./win32/configure.js
     default_options = {'shared': False,
                        'fPIC': True,
+                       'include_utils': False,
```

Comment:
```suggestion
                       'include_utils': True,
```

I remember that I have worked on an application requiring xmllint in the past and we added the binary to the Bincrafters' recipe because of it. Why did you remove it @SSE4? 

In most recipes we ship all library/executable options by default, so I'm thinking we should enable it by default here also

Reply:
looking at the diff:
```
    options = {"shared": [True, False],
               "fPIC": [True, False],
               "zlib": [True, False],
               "lzma": [True, False],
               "iconv": [True, False],
               "icu": [True, False]}
```
that's options we had before https://github.com/conan-io/conan-center-index/commit/16739f13e00cdc9527b108d549816df4a46032a1. need to blame further, I don't remember intentionally removing it.